autoscheduler/frontend/src/tests/ui/CourseSelectCard.test.tsx

```diff
@@ -65,244 +66,6 @@ describe('Course Select Card UI', () => {
     document.getElementsByTagName('html')[0].innerHTML = '';
   });
 
-  describe('groups sections by professor name and honors status, then orders by section number', () => {
```

Comment:
Got some linting warnings in this file that need to be fixed

Reply:
Got it thanks, didn't notice them lol.